flink-runtime/src/test/java/org/apache/flink/runtime/state/SharedStateRegistryTest.java

```diff
@@ -98,6 +106,77 @@ public void testUnregisterWithUnexistedKey() {
         sharedStateRegistry.unregisterUnusedState(Long.MAX_VALUE);
     }
 
+    @Test
+    public void testRegisterChangelogStateBackendHandles() throws InterruptedException {
+        SharedStateRegistry sharedStateRegistry = new SharedStateRegistryImpl();
+        long materializationId1 = 1L;
+        IncrementalStateHandleWrapper incrementalTMStateHandle1 =
+                createDummyIncrementalStateHandle(materializationId1);
+
+        IncrementalStateHandleWrapper incrementalJMStateHandle11 = incrementalTMStateHandle1.copy();
```

Comment:
NIT: To me, handle names aren't very informative and readable.
While reviewing, I renamed them to:
`matStateBase`
`matState1`
`nonMateState1`
`backendState1`
etc.
So that they are shorter and give convey information.
WDYT about such names?

Reply:
Already refactored.